README.md

```diff
@@ -0,0 +1,23 @@
+# NI JavaScript Style Guide
+
+Welcome to NI's internal and external JavaScript and TypeScript linter rules for [ESLint](https://eslint.org/docs/user-guide/getting-started) and [TSLint](https://palantir.github.io/tslint/).
+
+## Installation
+
+```
+npm install --save-dev eslint eslint-plugin-import @ni/eslint-config
```

Comment:
Doesn't need to block this PR but should we consider the approach used by [airbnb](https://www.npmjs.com/package/eslint-config-airbnb) using a command like:
```
npx install-peerdeps --dev @ni/eslint-config
```

I believe the benefit is we get to align our rules with specific versions of eslint, etc.. With the current approach new eslint, airbnb, etc versions could break someone trying to setup a new repo.

Reply:
Hmm, npm install would install the latest of eslint and eslint-plugin-import, wouldn't it? That command does seem helpful, although, you probably already know the syntax if you're using it. I will follow Airbnb's suit and include both a mention of the peer dependencies and the shortcut.

As for enforcing versions, I had the same concern. It really depends on how the upstream package's semantic versioning is handled, and [ESLint's recommendation](https://eslint.org/docs/developer-guide/shareable-configs#publishing-a-shareable-config) is to use the ```>=``` operator. This is likely to allow the use of the latest version most commonly used with a new setup, but [they follow semantic versioning](https://eslint.org/docs/maintainer-guide/releases)! They will break us across major releases, so how about we use the minor release operator, e.g. ```^7.6.0```, for eslint and eslint-plugin-import and the patch release operator for eslint-config-airbnb-base, e.g. ```~14.2.0```?

Reply:
I like the cut of your jib; sounds like a good approach to me 👍